python/LLM Examples/secure_prompt_from_file.py

```diff
@@ -0,0 +1,71 @@
+from privateai_client import PAIClient
+from privateai_client import request_objects
+import openai
+import sys
+import demo_config
+import argparse
+import json
+import base64
+import os
+
+# Initialize parser
+parser = argparse.ArgumentParser(description="Secure LLM prompting from a file")
+parser.add_argument("-d", "--directory", required=True, help="directory")
+parser.add_argument("-f", "--file", required=True, help="File to redact")
+parser.add_argument("-t", "--filetype", required=True, help="file type")
+parser.add_argument("-n", "--filename", required=True, help="file type")
+args = parser.parse_args()
+
+PRIVATEAI_API_KEY = demo_config.privateai["PROD_KEY"]
+PRIVATEAI_URL = demo_config.privateai["PROD_URL"]
+
+# Initialize the openai client
+openai.api_key = demo_config.openai["API_KEY"]
+
+file_dir = args.directory 
+file_name = args.filename 
+filepath = args.file 
+file_type= args.filetype.split("/")[1] 
+PRIVATEAI_SCHEME = 'https'
+client = PAIClient(PRIVATEAI_SCHEME, PRIVATEAI_URL)
+client.add_api_key(PRIVATEAI_API_KEY)
+
+
+def prompt_chat_gpt(text):
+    completion = openai.ChatCompletion.create(
+        model="gpt-4",
+        messages=[
+        {"role": "user",
+         "content": text}
+        ]
+    )
+    return completion.choices[0].message['content']
+
+# Read from file
+with open(filepath, "rb") as b64_file:
+    file_data = base64.b64encode(b64_file.read())
+    file_data = file_data.decode("ascii")
+
+# Make the request
+file_obj = request_objects.file_obj(data=file_data, content_type=args.filetype)
```

Comment:
change the content_type to be content_type if you wanna use the suggestion above